testutil/crypto/crypto.go

```diff
@@ -38,55 +46,42 @@ func NewCryptoIdentityFromIntSeed(i int) *CryptoIdentity {
 	return NewCryptoIdentityFromBytesSeed(seed)
 }
 
-func (v *CryptoIdentity) ABCIAddressBytes() []byte {
-	return v.SDKPubKey().Address()
-}
-
 func (v *CryptoIdentity) TMValidator(power int64) *tmtypes.Validator {
 	return tmtypes.NewValidator(v.TMCryptoPubKey(), power)
 }
 
 func (v *CryptoIdentity) TMProtoCryptoPublicKey() tmprotocrypto.PublicKey {
-	ret, err := sdkcryptocodec.ToTmProtoPublicKey(v.SDKPubKey())
+	ret, err := sdkcryptocodec.ToTmProtoPublicKey(v.ConsensusSDKPubKey())
 	if err != nil {
 		panic(err)
 	}
 	return ret
 }
 
 func (v *CryptoIdentity) TMCryptoPubKey() tmcrypto.PubKey {
-	ret, err := v.GetPubKey()
+	ret, err := sdkcryptocodec.ToTmPubKeyInterface(v.ConsensusSDKPubKey())
 	if err != nil {
 		panic(err)
 	}
 	return ret
 }
 
-func (v *CryptoIdentity) SDKStakingOperator() sdktypes.ValAddress {
-	return v.SDKStakingValidator().GetOperator()
-}
-
 func (v *CryptoIdentity) SDKStakingValidator() sdkstakingtypes.Validator {
-	ret, err := sdkstakingtypes.NewValidator(v.SDKValAddress(), v.SDKPubKey(), sdkstakingtypes.Description{})
+	ret, err := sdkstakingtypes.NewValidator(v.SDKValOpAddress(), v.ConsensusSDKPubKey(), sdkstakingtypes.Description{})
 	if err != nil {
 		panic(err)
 	}
 	return ret
 }
 
-func (v *CryptoIdentity) SDKPubKey() sdkcryptotypes.PubKey {
-	tmcryptoPubKey := v.TMCryptoPubKey()
-	ret, err := sdkcryptocodec.FromTmPubKeyInterface(tmcryptoPubKey)
-	if err != nil {
-		panic(err)
-	}
-	return ret
+func (v *CryptoIdentity) ConsensusSDKPubKey() sdkcryptotypes.PubKey {
+	return v.consensus.PubKey()
 }
```

Comment:
For consistency we could also define `OperatorSDKPubKey` to be used in `SDKValOpAddress`
```suggestion
}

func (v *CryptoIdentity) OperatorSDKPubKey() sdkcryptotypes.PubKey {
	return v.operator.PubKey()
}
```